internal/observability/events/event_type.go

```diff
@@ -0,0 +1,10 @@
+package event
+
+type Type string
+
+const (
+	EveryType       Type = "*"
```

Comment:
Opinions on changing EveryType to AllType(s)?

Reply:
I think EveryType (singular) reads better than AllType (singular) and all the type definitions are singulars.